lib/Models/Catalog/Esri/ArcGisFeatureServerCatalogItem.ts

```diff
@@ -257,8 +256,8 @@ class FeatureServerStratum extends LoadableStratum(
         return undefined;
       }
 
-      const source = new proj4.Proj((proj4definitions as any)[wkid]);
-      const dest = new proj4.Proj("EPSG:4326");
+      const source = (proj4definitions as any)[wkid];
```

Comment:
I was able to compile without this cast
```suggestion
      const source = proj4definitions[wkid];
```